IND/EconomicIndicators/EconomicIndicators.rdf

```diff
@@ -389,7 +390,7 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-ind-ei-ei;hasIndicatorValue"/>
-				<owl:onClass rdf:resource="&fibo-fnd-utl-alx;NumericIndex"/>
+				<owl:onClass rdf:resource="&fibo-fnd-utl-alx;NumericIndexValue"/>
```

Comment:
This seems wrong! The CPI for the US, for example, has many values, each for a different date. Is this meant to be the latest value - in which case the property name should indicate that e.g. hasLatestValue. 
In general case I think the reference should be from the values to the index (which you have), not vice versa.

Reply:
fixed

Reply:
You didn't address my 2nd point which is that the direction should be that the value should reference the Index; we should not encourage people to try to link from the Index to reference all its values over time.

Reply:
Let's do this in another issue - I think more work may be involved to clean some of this up.